index.html

```diff
@@ -0,0 +1,263 @@
+                    </div>
+                   
+                </div>
+                <div class="speaker-card">
+                    <img class="speaker-image" id="speak-img5" src="" alt="speaker-picture">
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name5"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali5">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info5">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+                <div class="speaker-card">
+                    <img class="speaker-image" id="speak-img6" src="" alt="speaker-picture">
+                    <div class="speaker-info">
+                        <h5 class="speaker-name" id="speak-name6"></h5>
+                        <p class="speaker-qualification main-color" id="speak-quali6">Risk analyst at International Fund for Agricultural Development</p>
+                        <hr class="speaker-hr">
+                        <p class="small-text" id="speak-info6">Omotola studies risks of investing in agricultural firms located in desert regions</p>
+                    </div>
+                   
+                </div>
+            </div>
+           
+            <div class="more-speakers">
+                <a class="show-more"  href="#"></a><button id="show-more" class="more-speakers-button">More <img class="angle-down" src="images/angle-down.png" alt="angle-down-icon"></button></a>
+            </div>
+        </div>
+    </section>
+    <section class="foot-section other-parts">
+        <div class="partners">
+            <h3 class="title-text move-foot">Partner<hr class="horizontal-line"></h3>
+            <div class="sponsor-logos">
+                <img class="sponsor-logo" src="images/spon-1.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-6.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-3.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-4.svg" alt="sponsor-logo">
+                <img class="sponsor-logo"  src="images/spon-5.svg" alt="sponsor-logo">
+            </div>
+
+        </div>
+        <div class="main-footer">
+            <img class="about-footer-image" src="images/my-logo.png" alt="summit-logo">
+            <div class="footer-info">
+                <h4>2023 Agricultural Enthusiast Nigeria. All Rights Reserved</h4>
+                <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Laudantium mollitia nisi, illo ratione voluptate ipsam perferendis cumque eaque inventore consequuntur aliquid at, aspernatur nesciunt corrupti minima expedita ea natus temporibus?</p>
+            </div>
+        </div>
```

Comment:
- [ ] The footer should be outside this section and it should be in the `<footer>` tag as it's a footer. Please kindly fix that in the about.html file 🙃